### ISSUE TITLE
[cli] no option exposed for forcing upgrades (e.g. helm --force)

### ISSUE THREAD

**Issue by ebauman** [rancher/rancher#25497]
**What kind of request is this (question/bug/enhancement/feature request):**
Enhancement/bug

**Steps to reproduce (least amount of steps as possible):**
`rancher app upgrade` does not offer a `--force` option

**Result:**
Cannot force upgrades. 

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): 2.3.5
- Installation option (single install/HA): HA

**Comment by dramich** [comment]
Available in v2.4.0-rc4

**Comment by rancher-max** [comment]
#### Validated using cli version v2.4.0-rc4
To test this, we need to be able to validate the request the rancher server sees when using this option in the cli command. Below are my steps:

1. Start rancher server with audit logs enabled: `sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 -v /var/log/rancher/auditlog:/var/log/auditlog -e AUDIT_LEVEL=3 rancher/rancher:v2.3.5`. Note that audit level can be 2 here instead.
2. Open the UI in a browser and add a cluster to it. I used a 1vcpu1GB linode cluster. 1 node, all 3 roles.
3. After cluster comes up, tail audit logs: `tail -f /var/log/rancher/auditlog/rancher-api-audit.log`
4. Create an app. I used the cli (in a new terminal window) to ensure the logs were showing me the data I wanted: `./rancher app install openebs -n openebs --version 1.5.0`
5. After the app has fully deployed and there is nothing more showing up in the tailed logs, run the upgrade command with the new option: `./rancher app upgrade -f <app-name> 1.6.0`

Ensure the logs (have to parse through them) contain the expected output:
```json
{
  "auditID": "b2542f61-a5f0-41d3-9ad8-f5899ee7955b",
  "requestURI": "/v3/project/c-xhbks:p-s5l2c/apps/p-s5l2c:a-8h4kc?action=upgrade",
  "user": {
    "name": "user-lq8tz",
    "group": [
      "system:authenticated"
    ]
  },
  "method": "POST",
  ...
  "requestBody": {
    "externalId": "catalog://?catalog=library&template=openebs&version=1.6.0",
    "forceUpgrade": true
  },
  "responseBody": null
}
```

Doing the upgrade through the UI, using the same force option, produces the same `requestBody.forceUpgrade` option. Therefore, I'd say this option is working as we'd expect.

I also did the same thing, leaving the option out, and it correctly resulted in that option not being present in the requestBody.
